### PR TITLE
HAWQ-1435 document new pxf jdbc plug-in

### PR DESCRIPTION
**Pull request by lisakowen**
document the community-contributed PXF JDBC plug-in.  include a simple mysql example.

### CODE REVIEW

**Comment by lisakowen** [comment]
todo - need to update the JDBC Custom Options table in this doc if HAWQ-1482 (pxf jdbc plugin empty RANGE values not implemented) is still pending.  also should verify there are no similar restrictions for the INTERVAL.

File: markdown/pxf/JdbcPXF.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+The JDBC plug-in `Jdbc` profile supports the following \<custom-option\>s:
+
+| Option Name   | Description
+|---------------|--------------------|
+| JDBC_DRIVER | The JDBC driver class name. |
+| DB_URL | The URL to the database; includes the hostname, port, and database name. |
+| USER | The database user name. |
+| PASS | The database password for USER. |
+| PARTITION_BY | The partition column, \<column-name\>:\<column-type\>. The JDBC plug-in supports `date`, `int`, and `enum` \<column-type\>s. Use the  `yyyy-MM-dd` format for the `date` \<column-type\>. A null `PARTITION_BY` defaults to a single fragment. |
+| RANGE | The query range, \<start-value\>[:\<end-value\>]. \<end-value\> may be empty for an `int` \<column-type\>. |
```
Comment (dyozie) [inline markdown/pxf/JdbcPXF.html.md.erb]: Should probably clarify that RANGE and INTERVAL are only used with PARTITION_BY?

File: markdown/pxf/JdbcPXF.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+| INTERVAL | The interval, \<interval-num\>[:\<interval-unit\>], of one fragment.  `INTERVAL` may be empty for an `enum` \<column-type\>. \<interval-unit\> may be empty for an `int` \<column-type\>. |
+
+**Note**: The partition `RANGE` interval is left closed, right open. That is, the range includes the \<start-value\> but does *not* include the \<end-value\>.
```
Comment (dyozie) [inline markdown/pxf/JdbcPXF.html.md.erb]: I'd move this (not as a note, necessarily, as MD formatting won't work) to the RANGE description in the table.

**Comment by lisakowen** [comment]
thanks for reviewing, @dyozie.  i have incorporated your comments.  let me know if i handled the required options appropriately.

**Pull request closed without merging**